go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/Azure/sonic-mgmt-common v0.0.0-00010101000000-000000000000
 	github.com/Workiva/go-datastructures v1.0.50
 	github.com/agiledragon/gomonkey/v2 v2.8.0
+	github.com/fortytw2/leaktest v1.0.0
```

Comment:
If the commented lines in the file above will be deleted then this line is not needed too, right?